vhdbuilder/packer/install-dependencies.sh

```diff
@@ -238,8 +238,8 @@ start_watch
 
 if [[ $OS == $UBUNTU_OS_NAME && $(isARM64) != 1 ]]; then  # no ARM64 SKU with GPU now
   gpu_action="copy"
-  NVIDIA_DRIVER_IMAGE_SHA="sha-ff213d"
-  export NVIDIA_DRIVER_IMAGE_TAG="cuda-535.54.03-${NVIDIA_DRIVER_IMAGE_SHA}"
+  NVIDIA_DRIVER_IMAGE_SHA="sha-2d4c96"
+  export NVIDIA_DRIVER_IMAGE_TAG="cuda-550.54.15-${NVIDIA_DRIVER_IMAGE_SHA}"
```

Comment:
why CUDA ? in gpu.go you are only creating the Grid variable, is this normal ?

nvidia550GridDriverVersion = "grid-550.54.15"

Reply:
CUDA is the driver to be updated. Removed the GRID change here - might make it in a separate PR as those are failing in agentbaker e2e too.